atomic_reactor/utils/cachito.py

```diff
@@ -225,13 +226,35 @@ def get_request_env_vars(self, rid):
                    f'but the response contains: {resp.content}.')
             raise ValueError(msg) from exc
 
-    def get_image_content_manifest(self, request_ids):
+    def get_request_config_files(self, rid: int) -> List[Dict[str, str]]:
+        """Get the configuration files from endpoint /requests/$id/configuration-files
+
+        :param int rid: the Cachito request id whose configuration files will be returned.
+        :return: a list of configuration files. For example:
+            ``[{"path": "app/.npmrc", "type": "base64", "content": "<base64 encoded content>"}]``.
+        :rtype: list[dict[str, str]]
+        :raises: ValueError if the server does not response a JSON data to be
+            parsed.
+        :raises: HTTP error raised from the underlying requests library in any
+            case of the request cannot be completed.
+        """
+        endpoint = f'{self.api_url}/api/v1/requests/{rid}/configuration-files'
+        resp = self.session.get(endpoint)
+        resp.raise_for_status()
+        try:
+            return resp.json()
+        except Exception as exc:
```

Comment:
is it possible we can qualify this exception?

Reply:
The other methods in this module also catch bare `Exception` :disappointed: 

I'm fairly certain that catching ValueError would be good enough, but I'd rather not accidentally change the behavior